src/client/components/MobileMsg.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export const MobileMsg: React.FC = () => {
+  return (
+    <div className="mobile-msg v-center" data-testid="mobile-msg">
+      <div className="text-center">
+        <h2>Please view screen on larger device</h2>
```

Comment:
Would it be best to be explicit and tell them that the application currently only runs on desktops/laptops? If I am a new user I might try an iPad (which would be cool to support btw) only to find that doesn't work either.

Reply:
`Please view screen on larger device, app currently only runs on iPad, desktops and laptops`. is OK for you? @kodyclemens. Btw, trying to create test escenario, on browser **Chrome** to **iPad** and **iPad Pro** and i've confirmed that the support is **OK**, view details:

![iPad-Details](https://user-images.githubusercontent.com/24376900/97085172-b731f580-15e9-11eb-8a73-c281dcb0ba4e.gif)